pages/signin.tsx

```diff
@@ -1,66 +1,98 @@
 import { useState } from 'react';
-import type { NextPage } from 'next';
+import Image from 'next/image';
 import Head from 'next/head';
+import { type NextPage } from 'next';
 import { supabase } from '../utils/supabaseClient';
+import { useRouter } from 'next/router';
 
 const SignIn: NextPage = () => {
-  const [email, setEmail] = useState<string>('');
-  const [password, setPassword] = useState<string>('');
-
+  const router = useRouter();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
   const handleSignIn = async () => {
     const { user, error } = await supabase.auth.signIn({ email, password });
     if (error) alert(error.message);
-    else alert(`Success Login in ${user?.email}`);
+    else {
+      alert(`Success Login in ${user?.email}`);
+      setEmail('');
+      setPassword('');
+    }
   };
-
   return (
-    <div className='flex min-h-screen flex-col items-center justify-center py-2'>
+    <>
       <Head>
-        <title>Sign In</title>
-        <link rel='icon' href='/favicon.ico' />
+        <title>Defaang / Welcome</title>
       </Head>
-      <div className='header'>Defaang</div>
-      <p>Sign in now!</p>
-      <form
-        onSubmit={e => {
-          e.preventDefault();
-          handleSignIn();
-        }}
-      >
-        <div className='w-50 mt-5'>
-          <label className='mb-2 block text-sm font-bold text-gray-700' htmlFor='email'>
-            Username
-          </label>
-          <input
-            className='focus:shadow-outline w-full appearance-none rounded border py-2 px-3 leading-tight text-gray-700 shadow focus:outline-none'
-            id='email'
-            type='email'
-            placeholder='E-Mail'
-            value={email}
-            onChange={e => setEmail(e.target.value)}
-          />
+      <main className='flex h-screen w-screen'>
+        {/* Left section - main login  */}
+
+        <div className='flex h-screen  w-[100%] flex-col items-center justify-center bg-gray-100 lg:w-[50%]'>
+          {/* Now you can decide different login ways - primarily I am giving google and twiiter for more add it yourself. */}
+          <div className='w-full  bg-white p-4 md:shadow-lg lg:h-[500px] lg:w-[400px]'>
+            <h1 className='p-2 text-3xl font-semibold'>Login ⚡</h1>
```

Comment:
let's use *Sign in* instead of *Login*

Reply:
Should I also do the same to create an account

Reply:
Create an account is fine, I think?